torchci/pages/api/drci/drci.ts

```diff
@@ -808,6 +810,29 @@ export async function getWorkflowJobsStatuses(
         continue;
       }
 
+      const matchDisabledTestIssues = getDisabledTestIssues(
+        job,
+        disabledTestIssues
+      );
+      if (
+        matchDisabledTestIssues !== undefined &&
+        matchDisabledTestIssues.length !== 0 &&
+        isRecentlyCloseDisabledTest(
+          matchDisabledTestIssues,
+          prInfo.merge_base_date
+        )
+      ) {
+        flakyJobs.push(job);
```

Comment:
Is there value in adding a check for `isDisabledTestMentionedInPR` in this code block?

Reply:
It's weird to try to fix a closed issue, but I agree that having `isDisabledTestMentionedInPR` here is safer and simpler, i.e. it doesn't matter what the issue is open or close, it won't be marked as flaky if it fails and is mentioned in the PR/commits